src/catalog/default/default_functions.cpp

```diff
@@ -163,6 +162,7 @@ static const DefaultMacro internal_macros[] = {
 
 	// date functions
 	{DEFAULT_SCHEMA, "date_add", {"date", "interval", nullptr}, {{nullptr, nullptr}}, "date + interval"},
+	{DEFAULT_SCHEMA, "make_timestamp_ms", {"milliseconds", nullptr, nullptr}, {{nullptr, nullptr}}, "make_timestamp(milliseconds * 1000::BIGINT)"},
```

Comment:
Since we still have `epoch_ms`, maybe we can make it an alias for that function instead?

Reply:
I meant to have both functions use the same C++ code - not to make the macro alias to `epoch_ms`. Currently `epoch_ms` will be more performant than `make_timestamp_ms` despite doing the same thing.